guides/common/modules/proc_benchmark-disk-performance.adoc

```diff
@@ -1,36 +1,41 @@
-This test will create a very large file that is double (2x) the size of the physical RAM on this system to ensure that we are not just testing the caching at the OS level of the storage.
-* Please bear above in mind when performing benchmark of other filesystems if you have them (like PostgreSQL storage) which might have significantly smaller capacity than Pulp storage and perhaps on different set of storage (SAN, iSCSI, etc).
+* One may have to temporarily reduce the RAM in order to run the I/O benchmark.
+For example if the host has 256 GB RAM, test would require 512 GB of storage to run.
+So as a workaround, you can add `mem=20G` kernel option in grub during system boot to temporary reduce size of system RAM.
+Test will create a very large file that is double (2x) the size of the RAM in specified directory and execute a series of storage I/O tests against it.
+The size of the file is needed to ensure that the test is not just testing the filesystem caching.
+* Please bear above in mind when performing benchmark of other filesystems.
+For example if you plan to test on smaller volumes (typically PostgreSQL storage), you might need to reduce RAM size as described above.
+* If you are using different storage solutions (SAN, iSCSI, and so on), they might be giving different performance.
+* We recommend you to stop all services before executing this script and you will be prompoted to do so.
 ====
 
-This test does not use directio and will utilize the OS plus caching as normal operations would.
+This test does not use direct I/O and will utilize file caching as normal operations would.
 
 // Would it make sense to package this script for Foreman?
 You can find our first version of the script https://github.com/RedHatSatellite/satellite-support/blob/master/storage-benchmark[storage-benchmark].
-To execute just download to your {Project}, `chmod +x` the script and run:
+To execute it, just download the script to your {Project}, `chmod +x` it and run:
 
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # ./storage-benchmark /var/lib/pulp
 ----
 
-As noted in the README block in the script: generally you wish to see on average 100MB/sec or higher in the tests below:
+As noted in the README block in the script, generally you wish to see on average 100MB/sec or higher in the tests below:
 
-* Local SSD based storage should values of 600MB/sec or higher.
-* Spinning disks should see values in the range of 100-200MB/sec or higher.
+* Local SSD based storage should give values of 600MB/sec or higher.
+* Spinning disks should give values in the range of 100-200MB/sec or higher.
 
 If you see values below this, please open a support ticket for assistance.
 
 ifndef::orcharhino[]
-For more information, see https://access.redhat.com/solutions/3397771[Impact of Disk Speed on {Project} Operations].
+For more information, see https://access.redhat.com/solutions/3397771[Impact of Disk Speed on Satellite Operations].
```

Comment:
You cannot use downstream terminology.

`{Project}` resolves to "Satellite" for Satellite documentation: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L71

Alternatively, you can change from "if not orcharhino" to "if satellite" to exclude the link from other build variants.

Reply:
Hello. This is just name of the article. Article will have "Satellite" in it's name even if the documentation will be built for upstream Foreman. If you think, I'm happy to change to `if satellite`. What would be your preference?

Reply:
Making it "satellite only" sounds good to me!

Reply:
Ack!